.pre-commit-config.yaml

```diff
@@ -0,0 +1,59 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+
+default_install_hook_types: [pre-push]
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v5.0.0
+    hooks:
+      - id: check-case-conflict
+      - id: check-docstring-first
+      - id: check-executables-have-shebangs
+      - id: check-json
+      - id: check-shebang-scripts-are-executable
+      - id: check-merge-conflict
+      # - id: check-symlinks
+      - id: check-toml
+      - id: check-xml
+      - id: check-yaml
+      - id: debug-statements
+      - id: destroyed-symlinks
+      - id: detect-private-key
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+      - id: fix-byte-order-marker
+      - id: mixed-line-ending
+      - id: requirements-txt-fixer
+      - id: trailing-whitespace
+
+  - repo: https://github.com/ansible/ansible-lint
+    rev: v25.1.0
+    hooks:
+      - id: ansible-lint
+        additional_dependencies:
+          - ansible
+
+  - repo: local
+    hooks:
+      # Needs `apt install shellcheck`.
+      - id: shellcheck
+        name: shellcheck
+        description: Static analysis tool for shell scripts
+        language: system
+        entry: shellcheck
+        args: [-f, gcc]
+        types: [shell]
+
+      # Needs `apt install shfmt`.
+      - id: shfmt
+        name: shfmt
```

Comment:
Final comment: perhaps run shfmt before shellcheck?

Other than the above, LGTM/thanks!